statemachine/src/test/java/com/amazonaws/stepfunctions/cloudformation/statemachine/CreateHandlerTest.java

```diff
@@ -268,6 +283,86 @@ public void testDefinitionFromS3InYAML() throws Exception {
         assertThat(response.getResourceModel().getDefinitionString()).isEqualTo(formattedJson);
     }
 
+    @Test
+    public void testNonStringDefinitionSubstitutionFromS3Yaml() throws Exception {
```

Comment:
Let's drop the 'NonString' part here. It makes it seem special compared to other acceptable substitution types.

Reply:
Will remove it in my next revision